src/shaders/symbol_icon.vertex.glsl

```diff
@@ -1,8 +1,15 @@
-attribute vec2 a_pos;
-attribute vec2 a_offset;
+
+#ifdef MAPBOX_GL_NATIVE
+  attribute vec4 a_pos_offset;
+#else
+  attribute vec2 a_pos;
+  attribute vec2 a_offset;
+#endif
+
```

Comment:
Should we just switch to packed representation for both JS & Native at the same time instead of adding an `ifdef`?